keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -205,12 +261,20 @@ the uses and implications of any populators they chose to install.
 
 ### Test Plan
 
-The minimal test for this feature is to create a PVC with a data source
-that's not a PVC or snapshot, and verify that the data source reference
+The test for this feature gate is to create a PVC with a data source
+that's not a PVC or VolumeSnapshot, and verify that the data source reference
 becomes part of the PVC API object. Any very simple CRD would be okay
 for this purpose. We would expect such a PVC to be ignored by existing
 dynamic provisioners.
 
+To test the data-source-validator, we need to check the following cases:
+- Creation of a PVC with no datasource is ignored
+- Creation of a PVC with a VolumeSnapshot or PVC datasource is ignored
+- Creation of a PVC with a CRD datasource that's not registered by any
+  volume populator causes events.
+- Creation of a PVC with a CRD datasource that's registered by a
+  volume populator is ignored.
```

Comment:
Same comment as before.  "is ignored" sounds negative.  Can we say something like "is accepted".